kripeshr22/MappingDemo
# Ynamgung/speed up inserts

Was able to speed up import speed from appx 56 hours per 1 million rows to appx 20 min per 1 million rows. Refactored the import_to_heroku and import_to_local scripts to run on batch imports

To test:

run the import_to_heroku.py file by typing into the console
>> python
>> import import_to_heroku
>> import_to_heroku.main()

To test the rate at which records are being inserted into the Heroku database check the table's record count (default table is "testtable"). The query for this is "select count(*) from testtable;"

To test import_to_local.py, repeat the same processes as for import_to_heroku.py except replace password/username/databasename/tablename if necessary. (reminder: database name is case sensitive)
